fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadJob.java

```diff
@@ -1056,7 +1062,11 @@ public void replayOnAborted(TransactionState txnState) {
             failMsg = new FailMsg(FailMsg.CancelType.LOAD_RUN_FAIL, txnState.getReason());
```

Comment:
better to override the function in `BrokerLoadJob` to correspond with `afterAborted`